libs/cart/src/actions/cart-payment.actions.ts

```diff
@@ -50,6 +54,43 @@ export class DaffCartPaymentUpdateFailure implements Action {
   constructor(public payload: string) {}
 }
 
+/**
+ * Triggers an update of the cart's selected payment method and billing address.
+ *
+ * @param payment The payment method.
+ * @param address The billing address.
+ */
+export class DaffCartPaymentUpdateWithBilling<
+  T extends DaffCartPaymentMethod = DaffCartPaymentMethod,
+  R extends DaffCartAddress = DaffCartAddress
+> implements Action {
+  readonly type = DaffCartPaymentActionTypes.CartPaymentUpdateWithBillingAction;
+
+  constructor(public payment: Partial<T>, public address: Partial<R>) {}
```

Comment:
Unless there's a reason for the payment type to be a Partial, I think it should be just `payment: T`

Reply:
oh, the interface for the driver takes a Partial<T>. Nevermind